CONTRIBUTING.md

```diff
@@ -23,6 +23,8 @@ You need to install the following dependencies:
   To install a specific Rust version, use `rustup install 1.32.0` command.
   * The [system dependencies](https://exonum.com/doc/version/0.10/get-started/install/) of Exonum. 
   You do _not_ need to manually fetch and compile Exonum.
+  * For automatic packaging of the EJB App you also need `CMake` installed in your system. Also on 
```

Comment:
"you also need ... Also you need ..."
I do not repeat twice twice 😄

Reply:
Please add a link for `CMake` as It done for maven, JDK and rust

Reply:
Fixed